index.bs

```diff
@@ -939,6 +939,34 @@ while (true) {
 }
 </pre>
 
+## Receiving from ReceiveStream ##  {#example-receiving-from-receivestream}
+
+*This section is non-normative.*
+
+Reading from ReceiveStreams can be achieved by calling
+{{UnidirectionalStreamsTransport/receiveStreams()}} method, then getting the
+reader for each {{ReceiveStream}}.
+
+<pre class="example" highlight="js">
+const transport = getTransport();
+const receiveStreamReader = transport.receiveStreams().getReader();
+while (true) {
+  const {value: receiveStream, done} = await receiveStreamReader.read();
+  if (done) {
+    break;
+  }
+  // Process ReceiveStreams created by remote endpoint.
+  const chunkReader = receiveStream.readable.getReader();
+  while(true){
```

Comment:
Please put a space between `while` and `(true)`. Also between `if` and `(done)` below.

Reply:
Fixed. Thanks.